src/printer.js

```diff
@@ -2342,6 +2354,7 @@ function typeIsFunction(type) {
 
 function printExportDeclaration(path, options, print) {
   const decl = path.getValue();
+  const semi =  options.noSemi ? "" : ";"
```

Comment:
We used to do format:all after each release since the number of PRs was near 0 but we skipped doing that during the last one and had a bunch of changes. It would be greatly needed indeed!